doc/docs/modules/ROOT/pages/quickstart.adoc

```diff
@@ -189,6 +207,36 @@ Reading data from a Neo4j Database can be done in 3 ways:
  * with a set of node Labels 
  * by specifying a relationship
 
+=== Considerations about partitioning
+
+While we're trying to pull off the data we offer the possibility to partition the extraction in order
+parallelizing it.
+
+==== How we parallelize the query execution
+
+Considering that we have three options
+
+1. Node extraction
+2. Relationship extraction
+3. Query extraction
+
+We adopt generally provide a general count on what you're trying to pull of and add build
+a query with the skip/limit approach over each partition.
+While for (1) and (2) we leverage the Neo4j count store in order to retrieve the total count
+about the nodes/relationships we're trying pulling off, for the (3) we have two possible approaches:
+
+* Compute a count over the query that we're using
```

Comment:
How would this be done in the general query case?  Or it can't be done, and using more than > 1 partition with a query where no count is specified will fail?